salt/modules/linux_sysctl.py

```diff
@@ -181,6 +166,9 @@ def persist(name, value, config=None):
     edited = False
     # If the sysctl.conf is not present, add it
     if not os.path.isfile(config):
+        sysctl_dir = os.path.split(config)[0]
```

Comment:
``os.path.dirname()`` is a bit quicker here, can you make this small performance tweak?

```python
>>> t1 = Timer('os.path.dirname("/etc/sysctl.d/99-salt.conf")', 'import os')
>>> t2 = Timer('os.path.split("/etc/sysctl.d/99-salt.conf")[0]', 'import os')
>>> t1.repeat()
[0.7001540660858154, 0.6916110515594482, 0.6758401393890381]
>>> t2.repeat()
[0.8055858612060547, 0.7713699340820312, 0.7606489658355713]
```